meson.build

```diff
@@ -6,7 +6,7 @@ project(
   # tools/version_utils.py
   version: '1.10.0.dev0',
   license: 'BSD-3',
-  meson_version: '>= 0.64.0',
+  meson_version: '>= 0.64.0,< 0.66',
```

Comment:
Unfortunately you cannot do this. I ran into that once before. It causes this in CI:
```
meson.build:1:0: ERROR: Meson version is 0.64.0 but project requires >= 0.64.0,< 0.66
```